src/components/composer/composer/composer.ts

```diff
@@ -273,6 +294,21 @@ export class Composer extends Component<Props, SpreadsheetChildEnv> {
   private processEnterKey(ev: KeyboardEvent) {
     ev.preventDefault();
     ev.stopPropagation();
+    if (ev.altKey || ev.ctrlKey) {
+      const selection = this.contentHelper.getCurrentSelection();
+      const currentContent = this.env.model.getters.getCurrentContent();
+      const content =
+        currentContent.slice(0, selection.start) + NEWLINE + currentContent.slice(selection.end);
+      this.env.model.dispatch("SET_CURRENT_CONTENT", {
+        content,
+        selection: { start: selection.start + 1, end: selection.start + 1 },
+      });
+
+      this.processContent();
+      this.contentHelper.scrollSelectionIntoView();
```

Comment:
I think it's slighly better to add a `scrollSelectionIntoView` into the `onMounted`. The problem with having it implied in the `processContent` is that if you scroll manually in the composer, you'd go back to the line of the cursor at each render. This isn't a huge problem in single-user, but it may be really annoying in multi-user.

Reply:
How could the multi user be impacted ? the content of the composer (while editing) is user exclusive

Reply:
Yeah but unless i'm wrong, if someone else dispatches a command, it will trigger a re-renders, thus new processContent and new scrollSelectionIntoView

Reply:
processcontent will fetch composerContent , which is either the content of*your* composer while you are editing, or the content of the cell, which inded might have been changed by someone else. So, while you are editing, you are not impacted by the changes done by others, except of course if the delete the cell you are editing :D

Reply:
Problematic use case : 
- you srcoll on your composer
- someone sends an unrelated command, this will trigger a re-render
- the re-render will trigger a processContent => a scrollIntoView => you lose your scroll